src/test/java/org/mdbenefits/app/pdf/PdfServiceTest.java

```diff
@@ -25,38 +27,46 @@ class PDFServiceTest {
-            .with("programs[]", List.of("SNAP", "RCA"))
-            .with("isApplicantPregnant", "false")
-            .with("signature", "John Hancock")
-            .with("someoneInHHIsPregnant", "true")
-            .with("someoneHasDrugKingpinFelony", "false")
-            .with("someoneHasVolumeDrugDealerFelony", "true")
-            .with("someoneHasSexualOffenceConviction", "false")
-            .with("someoneIsViolatingParole", "true")
-            .with("someoneConvictedForLyingAboutBenefits", "false")
-            .with("someoneConvictedForTradingBenefits", "true")
-            .with("someoneIsReceivingBenefitsWithFakeID", "false")
-            .build();
+                .withPersonalInfo("John", "Doe", "10", "12", "1999",
+                        "", "", "", "", "111111111")
+                .withHouseholdMember("Jane", "Doe", "1", "5", "2000",
+                        "wife", "F", "", "", "222222222", null, null)
+                .with("homeAddressStreetAddress1", "972 Mission St, 5th Floor")
+                .with("homeAddressCity", "San Francisco")
+                .with("homeAddressState", "CA")
+                .with("homeAddressZipCode", "94103")
+                .with("phoneNumber", "(510) 555-1212")
+                .with("programs[]", List.of("SNAP", "RCA"))
+                .with("isApplicantPregnant", "false")
+                .with("signature", "John Hancock")
+                .with("someoneInHHIsPregnant", "true")
+                .with("someoneHasDrugKingpinFelony", "false")
+                .with("someoneHasVolumeDrugDealerFelony", "true")
+                .with("someoneHasSexualOffenceConviction", "false")
+                .with("someoneIsViolatingParole", "true")
+                .with("someoneConvictedForLyingAboutBenefits", "false")
+                .with("someoneConvictedForTradingBenefits", "true")
+                .with("someoneIsReceivingBenefitsWithFakeID", "false")
+                .with("applicantRace[]", List.of(RaceType.ASIAN.name(), RaceType.WHITE.name()))
+                .build();
         submission.setFlow("mdBenefitsFlow");
 
         File pdfFile = pdfService.generate(submission);
-        String text = getText(pdfFile);
-        assertThat(text).contains("Doe, John");
+
+        String page1 = getPageText(pdfFile, 1);
+        assertThat(page1).contains("Doe, John");
+
+        String page2 = getPageText(pdfFile, 2);
+        // applicant DOB
+        assertThat(page2).contains("12/10/1999");
+        // applicant Race
+        assertThat(page2).contains("2,5");
     }
 
-    private static String getText(File file) throws IOException {
+    private static String getPageText(File file, int page) throws IOException {
         try (PdfReader reader = new PdfReader(file.getPath())) {
             PdfTextExtractor pdfTextExtractor = new PdfTextExtractor(reader);
-            return pdfTextExtractor.getTextFromPage(1);
+            return pdfTextExtractor.getTextFromPage(page);
         }
```

Comment:
Not crazy about the time it takes to run the extractor, just to extract a page at a time, but I can't think of a better way to do this right now and it's probably not a big hit anyways.